packages/api/lib/reconciliationReport.js

```diff
@@ -117,6 +117,36 @@ function convertToDBGranuleSearchParams(params) {
   return removeNilProperties(searchParams);
 }
 
+/**
+ *
+ * @param {Object} params - request params to convert to database params
+ * @returns {Object} object of desired parameters formated for database granule search
+ */
+function convertToDBScanGranuleSearchParams(params) {
+  const {
+    collectionId,
+    granuleId: granuleIdParam,
```

Comment:
There won't be `granuleId` parameter after normalizeEvent, only `granuleIds`

Reply:
Ahhh yeah I see that now. Thanks, will adjust

Reply:
Can you add a test case for the lambda `packages/api/tests/lambdas/test-create-reconciliation-report.js`?

Reply:
Absolutely!